### PR TITLE
Add consts/text for Gen 9 abilities

### PR DESCRIPTION
**Pull request by Drayano60**
Some of these abilities have too many effects to squeeze into two lines <__>

(Assuming it's fine to add these, if you want to keep Gen 9 stuff out for now dw)

### CODE REVIEW

File: data/text/720.txt [hunk 1]
```diff
@@ -265,4 +265,35 @@ Dragon’s Maw
 Chilling Neigh
 Grim Neigh
 As One
-As One
\ No newline at end of file
+As One
```
Comment (Drayano60) [inline data/text/720.txt]: I'm not sure what's with the diff here tbh

**Comment by Drayano60** [comment]
After some experimentation this is probably a bit premature; there's still only one byte for each ability in the personal data I think so abilities with an ID of 256 or over don't work

**Comment by BluRosie** [comment]
yeah, need to edit the GetBoxMonData function to grab the ability from either a different field or add one of the bits from the same byte we put the hidden ability thing

**Comment by BluRosie** [comment]
that would probably still have bugs with fields expecting a u8 so that'd be a second i believe

**Pull request merged**